guides/source/getting_started.md

```diff
@@ -71,9 +71,8 @@ The Rails philosophy includes two major guiding principles:
 Creating a New Rails Project
 ----------------------------
 
-The best way to use this guide is to follow each step as it happens, no code or
-step needed to make this example application has been left out, so you can
-literally follow along step by step.
+The best way to read this guide is to follow it step by step. All steps are important 
```

Comment:
Can you wrap your additions at 80 chars and remove the whitespace at the end of the line please ?